pkg/storkctl/action.go

```diff
@@ -40,10 +43,13 @@ func newFailoverCommand(cmdFactory Factory, ioStreams genericclioptions.IOStream
 				util.CheckErr(err)
 				return
 			}
+			var actions []storkv1.Action
 			for _, namespace := range namespaces {
 				if anyActionIncomplete(namespace) {
 					printMsg(
-						fmt.Sprintf("Cannot perform new action as an action is already scheduled/in-progress for %v", namespace),
+						fmt.Sprintf(
+							"failed to start failover for namespace %v as an action is already in scheduled/in-progress",
```

Comment:
nit: I would say lets print the status here, so that in future if we add more statuses we don't need to change the text.